### PR TITLE
ukrainian localization

### PR DESCRIPTION
**Pull request by web-ruslan**
ukrainian localization fix

### CODE REVIEW

**Review by web-ruslan — COMMENTED** [review]
Review needed @zaaom @fletch3555 @marktopper 
Thank You for help!

File: publishable/lang/uk/database.php
Comment (zaaom) [inline publishable/lang/uk/database.php:32]: have been away from the repo for a while. but why these keys are removed?

Comment (web-ruslan) [reply publishable/lang/uk/database.php:32]: @zaaom these keys are removed from english version https://github.com/the-control-group/voyager/blob/1.6/publishable/lang/en/database.php

**Review by zaaom — APPROVED** [review]

**Comment by web-ruslan** [comment]
@zaaom still review is required :(

**Comment by web-ruslan** [comment]
Review needed @zaaom  @fletch3555  @marktopper 
Thank You for help!

**Comment by zaaom** [comment]
@web-ruslan you need a review from a maintainer to merge.

**Comment by web-ruslan** [comment]
Review needed @emptynick 
Thank You for help!

**Review by zaaom — APPROVED** [review]

**Comment by web-ruslan** [comment]
@tnylea please check this code and merge it

**Pull request closed without merging**
